sdcm/nemesis.py

```diff
@@ -1568,7 +1568,8 @@ def disrupt_no_corrupt_repair(self):
                 for i in range(10):
                     time.sleep(random.randint(0, 300))
                     with self.cluster.cql_connection_patient(self.target_node) as session:
-                        session.execute(f'DROP TABLE drop_table_during_repair_ks_{i}.standard1')
+                        session.execute(SimpleStatement(
+                            f'DROP TABLE drop_table_during_repair_ks_{i}.standard1', fetch_size=1), timeout=300)
```

Comment:
I think you can omit the fetch_size, it's irrelevant for those cql commands

Reply:
removed